ddtrace/contrib/elasticsearch/transport.py

```diff
@@ -1,67 +0,0 @@
-# DEV: This will import the first available module from:
-#   `elasticsearch`, `elasticsearch1`, `elasticsearch2`, `elasticsearch5`, 'elasticsearch6'
-from .elasticsearch import elasticsearch
-
-from .quantize import quantize
-
-from ...constants import SPAN_MEASURED_KEY
-from ...utils.deprecation import deprecated
-from ...compat import urlencode
-from ...ext import SpanTypes, http, elasticsearch as metadata
-from ...settings import config
-
-DEFAULT_SERVICE = "elasticsearch"
-
-
-@deprecated(message="Use patching instead (see the docs).", version="1.0.0")
-def get_traced_transport(datadog_tracer, datadog_service=DEFAULT_SERVICE):
```

Comment:
feels weird to remove this now. I can't actually justify it, just feels weird, that is all.

Reply:
Yeah a bit of a cheat since it's not v1.0.0. There were a couple of test failures and I really didn't want to invest more effort supporting something that very few (if anyone) uses.

Reply:
I was torn over how we handle. Like, do we keep the function as a no-op but with the deprecation warning? Do we keep the function as a no-op, but have it raise a real exception saying to use patching? Or do we just remove entirely, which will raise an exception.

Providing some kind of feedback to the user is always good, but not sure I have a strong opinion here one way or the other.

Reply:
Yeah, I mean since we're 0.x (unstable) and we explicitly state in our docs that we're unstable and to check release notes when upgrading then I think we're safe to make these kind of changes. Keeping it as no-op arguably is just as wrong as now the instrumentation is going to silently not work which is also broken. I shift between these two approaches constantly but am starting to settle in the direction of failing loudly.